content/en/Requirements/Hardware/Design_requirements_outline.adoc

```diff
@@ -0,0 +1,254 @@
+
+---
+title: "Project Tetra Hardware Requirements Outline"
```

Comment:
I would personally re-title this "Hardware Requirements Outline" as to remove the redundancy